_scripts/generate-changelog.sh

```diff
@@ -40,7 +40,7 @@ main() {
    # if REPOROOT is already in the environment, don't overload it and use that
```

Comment:
Could you instead use the `generate-changelog.sh` from registry or logger? Those are slightly updated from this version and pass `shellcheck`.

Ah, the glory of copy-paste.